tools/hlint.sh

```diff
@@ -12,6 +12,9 @@ while getopts ':f:m:' opt
             if [ "$f" = "all" ]; then
               files=$(find libs/ services/ -not -path "*/test/*" -name "*.hs")
               echo "WARNING: not linting tests."
+            elif [ "$f" = "pr" ]; then
+              files=$(git diff --name-only develop | grep \.hs\$)
```

Comment:
```suggestion
              files=$(git diff --name-only origin/develop... | grep \.hs\$)
```
then you don't to update your local develop branch. a `git fetch` would be enough